packages/neuron-ui/src/services/remote/offline.ts

```diff
@@ -40,7 +40,8 @@ export type BroadcastProps = OfflineSignJSON & { walletID: string }
 
 export const exportTransactionAsJSON = remoteApi<OfflineSignJSON, void>('export-transaction-as-json')
 export const signTransactionOnly = remoteApi<OfflineSignJSON, void>('sign-transaction-only')
-export const broadcastTransaction = remoteApi<BroadcastProps, void>('broadcast-transaction-only')
+export const broadcastTransaction = remoteApi<BroadcastProps, void>('broadcast-transaction')
+export const broadcastTransactionOnly = remoteApi<OfflineSignJSON, string>('broadcast-transaction-only')
```

Comment:
How about typing it with `SignedTransaction` instead of `OfflineSignJSON` because `OfflineSignJSON` reminds me of `transaction to offline-sign`